Dockerfile

```diff
@@ -33,4 +33,5 @@ COPY . $WORKDIR
 RUN chgrp -R 0 $WORKDIR && \
     chmod -R g=u $WORKDIR
 
-ENTRYPOINT ["bin/openshift-collector"]
+ENTRYPOINT ["/usr/bin/haberdasher"]
```

Comment:
if you want to test operations worker on stage, you need to edit `deploy/clowdapp.yaml` this way:
```
podSpec:
  ...
  command:
  - /usr/bin/haberdasher
  args:
  - bin/openshift-operations
  ...
```

also follow the step 1 and download haberdasher in this file: https://github.com/RedHatInsights/haberdasher#adding-it-to-your-dockerfile

Reply:
not related to topo anymore